lighthouse-core/config/v2/config.js

```diff
@@ -0,0 +1,186 @@
+const path = require('path');
+const log = require('../../lib/log');
+
+const defaultConfigPath = './default.json';
+const defaultConfigJson = require(defaultConfigPath);
+
+const _flatten = arr => [].concat(...arr);
+const _subtract = (setA, setB) => Array.from(setA).filter(x => !setB.has(x));
```

Comment:
sure, and it did at first but it never gets used as a set, just as an array, I'll rename to `_differenceAsArray`